<launch>
	<!-- parameter -->
	<arg name="rosbag_dir" default="$(env HOME)/rosbag/airsim/drone_2cam_lidar16"/>
	<arg name="rosbag_name" default="2022-10-10-13-32-27_transformed"/>

	<!-- main -->
	<node pkg="pc_merge" type="pc_merge_offline" name="pc_merge_offline" output="screen">
		<param name="load_rosbag_path" type="string" value="$(arg rosbag_dir)/$(arg rosbag_name).bag"/>
		<!-- <param name="debug_hz" type="double" value="10.0"/> -->
		<param name="pc_0" type="string" value="/airsim_node/drone/lidar/LidarCustom"/>
		<param name="pc_1" type="string" value="/airsim_node/drone/lidar/LidarCustom/transformed"/>
	</node>

	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find pc_merge)/rviz_config/debug.rviz"/>
</launch>